syntaxes/starlark.tmLanguage.json

```diff
@@ -2,7 +2,10 @@
     "name": "Starlark",
     "scopeName": "source.starlark",
     "fileTypes": [
-        "bzl"
+        "bzl",
+        "star",
+        "build_defs",
```

Comment:
I don't think `build_defs` is used.

Reply:
In specific, https://please.build/ declares build_defs files in Starlark/bazel syntax